js/plugins/jsqueue-draggable.js

```diff
@@ -25,7 +25,23 @@ function jsqueue_draggable() {
 
     this.MAKE_SORTABLE = function (data) {
         var self = this;
+
+        if (data['file_upload_event']) {
+            data.options['receive'] = function(event, ui) {
+                var item_id = ui['item'].attr('id');
+
+                if (!event['target']['id'].startsWith('map-row')) {
```

Comment:
Same here lets make this generic and have the map-row come in via data so we can set it.